src/conn.c

```diff
@@ -866,14 +870,10 @@ void xmpp_send_raw_string(xmpp_conn_t *conn, const char *fmt, ...)
         xmpp_vsnprintf(bigbuf, len, fmt, ap);
         va_end(ap);
 
-        xmpp_debug(conn->ctx, "conn", "SENT: %s", bigbuf);
```

Comment:
Please, don't remove this debug message and below. I understand that it is not precisely correct since we even don't queue stanza at this point. However, some applications use SENT and RECV debug messages, for example `xmppconsole`. I guess I've seen other applications that rely on these messages.

Reply:
Oh OK, I wasn't aware of the specific syntax.

The statement wasn't removed but moved to `_send_raw()`. I will change it again to be prefixed by 'SENT:'